### PR TITLE
Fix minor text/styling issues in the profile creation pages

### PR DESCRIPTION
**Pull request by malaikaaiyar**
fixes https://github.com/sparkletown/internal-sparkle-issues/issues/821

except for #9 (which would require more work than I can do right now)

image

### CODE REVIEW

File: src/pages/Account/Questions.tsx [hunk 1]
```diff
@@ -58,24 +58,23 @@ const Questions: React.FunctionComponent<PropsType> = ({ location }) => {
...
           {venue.profile_questions &&
-            venue.profile_questions.map((question: QuestionType) => (
-              <div key={question.name} className="input-group">
+            venue.profile_questions.map((question: QuestionType, index) => (
+              <div key={question.name} className="input-group question-input">
+                {index <= 1 ? (
+                  <p> {question.name}* </p>
+                ) : (
+                  <p> {question.name} </p>
+                )}
```
Comment (sunny-viktoryia) [inline src/pages/Account/Questions.tsx:77]: would be great to make a const NUMBER_OF_REQUIRED_FIELDS = 2; and use it here.
```suggestion
                <p>
                  {question.name}
                  {index < 2 && "*"}
                </p>
```

**Review by sunny-viktoryia — COMMENTED** [review]
light wording improvements

File: src/pages/Account/Questions.tsx [hunk 2]
```diff
@@ -58,24 +58,23 @@ const Questions: React.FunctionComponent<PropsType> = ({ location }) => {
...
           {venue.profile_questions &&
-            venue.profile_questions.map((question: QuestionType) => (
-              <div key={question.name} className="input-group">
+            venue.profile_questions.map((question: QuestionType, index) => (
+              <div key={question.name} className="input-group question-input">
+                {index <= 2 ? (
+                  <p> {question.name}* </p>
+                ) : (
+                  <p> {question.name} </p>
+                )}
```
[the remark this answers is not in the document]
Comment (sunny-viktoryia) [reply src/pages/Account/Questions.tsx]: the comment was made earlier, and then after the latest changes couldn't find it to remove before the review submitting; so resolving it.

[the remark this answers is not in the document]
Comment (malaikaaiyar) [reply src/pages/Account/Questions.tsx]: yeah I fixed this

File: src/pages/Account/Profile.tsx [hunk 3]
```diff
@@ -66,36 +66,43 @@ const Profile: React.FunctionComponent<PropsType> = ({ location }) => {
     <div className="page-container-onboarding">
       <div className="login-container">
         <h2 className="login-welcome-title">
-          Well done! Now create your profile
+          Hey, Mona. We’re so glad you’re here! Upload or take a photo and share
+          your Summit snap here.
         </h2>
         <div className="login-welcome-subtitle">
-          {`Don't fret, you'll be able to edit it at any time later`}
+          {`Swing back and edit your profile anytime you like.`}
```
Comment (sunny-viktoryia) [inline src/pages/Account/Profile.tsx:73]: [nit] design
image

our version
image

looks like we need to give a bigger `margin-top` to the "Swing back..." text

File: src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx [hunk 4]
```diff
@@ -152,7 +152,9 @@ const ProfilePictureInput: React.FunctionComponent<ProfilePictureInputProps> = (
       )}
       {isPictureUploading && <small>Picture uploading...</small>}
       {error && <small>Error uploading: {error}</small>}
-      <small>Or pick one from our Sparkle profile pics</small>
+      <small>
+        Feeling camera shy? Choose from one of these Summit profile avatars.
+      </small>
```
Comment (sunny-viktoryia) [inline src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx:157]: [nit] styling
(the screenshot from the design)
image

our version
image


I don't think we need `<small >` here. Instead we need to increase margin and give `opacity: 0.8` to it.

File: src/pages/Account/Account.scss [hunk 5]
```diff
@@ -88,6 +94,10 @@ $page-max-width: 1240px;
     text-align: center;
     padding-bottom: 30px;
   }
+
+  .summit-snap {
```
Comment (sunny-viktoryia) [inline src/pages/Account/Account.scss:98]: [NB] we should use BEM-style class names in the file.

File: src/pages/Account/Account.scss [hunk 6]
```diff
@@ -124,6 +134,16 @@ $page-max-width: 1240px;
...
   }
+
+  &.github {
+    margin: 0 auto;
+    background-image: url(/assets/images/Summit_Logo.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 580px;
+    height: 225px;
+    background-size: 280px;
```
Comment (sunny-viktoryia) [inline src/pages/Account/Account.scss:145]: [NB] use scss variables

File: src/pages/Account/Questions.tsx [hunk 7]
```diff
@@ -58,24 +58,22 @@ const Questions: React.FunctionComponent<PropsType> = ({ location }) => {
...
-        </p>
         <form onSubmit={handleSubmit(onSubmit)} className="form">
           {venue.profile_questions &&
-            venue.profile_questions.map((question: QuestionType) => (
-              <div key={question.name} className="input-group">
+            venue.profile_questions.map((question: QuestionType, index) => (
+              <div key={question.name} className="input-group question-input">
+                <p>
+                  {question.name}
+                  {index < 2 && "*"}
```
Comment (sunny-viktoryia) [inline src/pages/Account/Questions.tsx:75]: [NB] a descriptive const for "2"

**Review by sunny-viktoryia — APPROVED** [review]
looks like it's a working solution. I have some comments about styling here, but taking into account that it's needed as soon as possible, it's more important to merge the functionality first and then (if time allows) to fix the styles. 

@malaikaaiyar would be cool if you could create a follow up PR that fixes the comments I mentioned here.

File: src/pages/Account/Account.scss (hunk 6 above)
Comment (malaikaaiyar) [reply src/pages/Account/Account.scss:145]: the .hero-logo and .sparkle styles don't use variables, so i thought we shouldn't

**Comment by sofisparkle** [comment]
https://github.com/sparkletown/internal-sparkle-issues/issues/821 *magic linking gestures*

**Comment by 0xdevalias** [comment]
@malaikaaiyar When resolving review comments, please reply in the comment threads, and link the commit hash where the comment was resolved. This helps the reviewer. You can read more about our standards on this in https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1620395110209200

**Pull request merged**